### DISCUSSION TITLE
V1：使用initialValues给Form赋值时ArrayTable组件会记住初始值

### DISCUSSION THREAD

**Discussion by wangxiaofeid** [alibaba/formily#1219] [Q&A 1.x] [answered]
demo：https://codesandbox.io/s/stoic-gates-1ovrn
问题：赋了初始值，删除一条记录后再添加，初始的赋值的还会被填入对应的输入框

**Comment by wangxiaofeid** [comment, marked as answer]
使用`initialValues `和`defaultValue `是类似情况，`value`是没问题的，是否可以改成value?

**Reply by janryWang** [reply]
这个是默认值的默认行为，改成value即可

**Reply by wangxiaofeid** [reply]
使用value的时候如果我页面有交互，比如点击某个按钮会触发ajax请求，然后我再表单外层添加了一层Spin，这就导致了表单重渲染，从而导致value值变成了老的，最新添加的值丢失。initialValues的这种默认行为能否可否有什么配置修改掉

**Reply by wangxiaofeid** [reply]
使用value的时候只有第一次修改后重渲染会冲掉新值，后面修改不会被冲掉，这个现象有点奇怪

**Reply by wangxiaofeid** [reply]
0.4.4版本的uform有这个问题，1.*的formily没这个问题

**Reply by janryWang** [reply]
升级1.x，0.x现在不维护了的
